src/sentry/static/sentry/app/components/stacktracePreview.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Event, Organization, PlatformType} from 'app/types';
+import {Client} from 'app/api';
+import withApi from 'app/utils/withApi';
+import Hovercard, {Body} from 'app/components/hovercard';
+import {isStacktraceNewestFirst} from 'app/components/events/interfaces/stacktrace';
+import StacktraceContent from 'app/components/events/interfaces/stacktraceContent';
+
+type Props = {
+  issueId: string;
+  organization: Organization;
+  api: Client;
+};
+
+type State = {
+  event: Event | null;
+};
+
+class StacktracePreview extends React.Component<Props, State> {
+  state: State = {
+    event: null,
+  };
+
+  fetchData = async () => {
+    if (this.state.event) {
+      return;
+    }
+
+    const {api, issueId} = this.props;
+    try {
+      const event = await api.requestPromise(`/issues/${issueId}/events/latest/`);
+      this.setState({event});
+    } catch {
+      // preview will not show up
+    }
+  };
+
+  handleStacktracePreviewClick = (event: React.MouseEvent) => {
+    event.stopPropagation();
+    event.preventDefault();
+  };
+
+  render() {
+    const {organization, children} = this.props;
+    const {event} = this.state;
+    const exception = event?.entries.find(e => e.type === 'exception')?.data;
+    const stacktrace = exception?.values[0]?.stacktrace;
+
+    if (!organization.features.includes('stacktrace-hover-preview')) {
```

Comment:
should we maybe verify the feature flag before calling this component?

Reply:
I was also considering it but with the way things are structured I'd rather do it like this 👍